doc/imv.1

```diff
@@ -1,131 +1,125 @@
-.TH IMV 1 "" "" "multimedia"
-.SH NAME
-imv \- image viewer
-.SH SYNOPSIS
-.nf
-\fBimv\fP [-rfaudlh] [-n <NUM|PATH>] [-b COLOR] [-e FONT:SIZE] [-t SECONDS] [images...]
-.fi
-.sp
-.SH DESCRIPTION
-.sp
-\fBimv\fP is a X11/Wayland image viewer aimed at users of tiling window managers.
+.Dd $Mdocdate: December 30 2015$
```

Comment:
Viewing the new manpage using man 2.7.5 displays the `$` after `2015`. I'm also seeing "BSD General Commands Manual" written to the header, and "BSD" added to the footer. Is this an assumption the man page reader is making owing to the syntax used?